Makefile

```diff
@@ -4,6 +4,7 @@
 PWD := $(shell pwd)
 USER_ID := $(shell id -u)
 GROUP_ID := $(shell id -g)
+TORCH_VERSION := 2.3.1
```

Comment:
what is it for ?

Reply:
It was an artifact from a previous commit, I removed it